lib/jekyll/data_file.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Jekyll
+  class DataFile
+    attr_accessor :context
+    attr_reader :data
+
+    def initialize(site, abs_path, data)
```

Comment:
A comment here would still be nice, since I have to make certain assumptions about these variables. Something like:

```ruby
# Create a DataFile.
# site - the Jekyll::Site this file belongs to
# abs_path - the absolute path to the file
# data - the parsed contents of the data file
```

If the comment is wrong, then it only helps to illustrate why a comment is helpful 😜

Reply:
Your assumptions here are correct. I'll add the suggested comment to avoid assumptions. 🙂